extensions/notebook/src/test/book/book.test.ts

```diff
@@ -533,6 +538,34 @@ describe('BooksTreeViewTests', function () {
 			should(showPreviewSpy.notCalled).be.true('Should not call showPreviewFile when showPreview isn\' true');
 		});
 
+		it('openMarkdown should open markdown in the editor', async () => {
+			let executeCommandSpy = sinon.spy(vscode.commands, 'executeCommand');
+			let notebookPath = path.join(rootFolderPath, 'content', 'readme.md');
+			bookTreeViewProvider.openMarkdown(notebookPath);
+			should(executeCommandSpy.calledWith('markdown.showPreview')).be.true('openMarkdown should have called markdown.showPreview');
+		});
+
+		// TODO: Need to investigate why it's failing on linux.
+		it.skip('openNotebook should open notebook in the editor', async () => {
```

Comment:
Why did you change this? Having unstable tests run in the unstable runs is potentially still useful so we should use that over skip when we can.

Reply:
Unstable tests should be unstable. I'm not convinced that tests that always fail should be run as part of "unstable"

Reply:
It fails 100% of the time? I mean to me if that's the case that sounds like a concern we should investigate now then...the test is testing a pretty basic scenario and if that isn't working we have some big issues

Reply:
Lemme try to help out here to unblock. Because it actually fails for me when i un-skip the test on my Mac.